src/test/java/bt/TestWalett.java

```diff
@@ -0,0 +1,37 @@
+package bt;
+
+import lombok.Data;
+import org.apache.commons.lang3.RandomStringUtils;
+import signumj.crypto.SignumCrypto;
+import signumj.entity.SignumAddress;
+import signumj.entity.SignumValue;
+import signumj.entity.response.Account;
+
+@Data
+public class TestWalett extends BT{
```

Comment:
TestAccount? (calling an account a "wallet" is a misconception - wallet is the software allowing to interact with the chain).

Reply:
I ll change walett to account